integrations/disco/hooks/use-disco-issue-credential.ts

```diff
@@ -0,0 +1,98 @@
+import { setTimeout } from 'timers'
+
+import { zodResolver } from '@hookform/resolvers/zod'
+import { useMutation } from '@tanstack/react-query'
+import { useForm } from 'react-hook-form'
+import { z } from 'zod'
+
+import { useToast } from '@/lib/hooks/use-toast'
+
+import { appDiscoPostCredentialIssue } from '../routes/post-credential-issue/client'
+
+export const useDiscoIssueCredential = () => {
+  const { toast, dismiss } = useToast()
+
+  const handleToast = (title: string, description: string) => {
+    toast({
+      title: title,
+      description: description,
+    })
+
+    setTimeout(() => {
+      dismiss()
+    }, 4200)
+  }
+  const mutation = useMutation(async (vars) => {
+    const res = await appDiscoPostCredentialIssue(vars)
+
+    return res
+  })
```

Comment:
Always use the object input form for clarity, and the mutationFn expects a simple promise returning function, so there's no need to use `async await`. Also this usemutation should be abstracted to a hook like the other hooks in the integration
```suggestion
  const mutation = useMutation({
    mutationFn: (vars) => {
      return appDiscoPostCredentialIssue(vars)
    },
  })
```